### ISSUE TITLE
Discussion regarding Title of demo

### ISSUE THREAD

**Issue by xadahiya** [HTTP-APIs/hydra-astrodemo#16]
Since we're giving the user an ability to connect to any hydra server they want we should use a dynamic title instead of a hardcoded one(Hydra Astrodemo) or if we're using the hardcoded titles then there shouldn't be the option to connect to other clients.
What does everyone else think? Please share your opinions here.

**Comment by Mec-iS** [comment]
If there is the possibility of making everything working in a more general
way ok. But the tutorial will be based on a particular server, isn't it? We
cannot make at this time a tutorial "template" that works with any server,
can't we?

Lorenzo Moriondo, from mobile
https://it.linkedin.com/in/lorenzomoriondo

**Comment by xadahiya** [comment]
Yeah, that's what I've been thinking. One solution is to keep everything dynamic but by default the demo is connected to the astrodemo server, giving the user option to disconnect if he wants to. Shouldn't this work?

**Comment by Mec-iS** [comment]
The tutorial should be based on a server chosen by default. Then the user could also explore other vocabularies.
